fdbserver/storageserver.actor.cpp

```diff
@@ -2209,6 +2238,7 @@ Optional<MutationRef> clipMutation(MutationRef const& m, KeyRangeRef range) {
 	return Optional<MutationRef>();
 }
 
+// Return whether or not the there is a mutation that need to be done.
```

Comment:
I don't quite follow this comment.

Reply:
+1

Reply:
Rephrased to
```
// Return true if the mutation need to be applied, otherwise (it's a CompareAndClear mutation and failed the comparison) false.
```